plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -34,6 +34,18 @@
     description:
     - The L3Out associated with the external epg.
     - Required when site is of type on-premise.
+    - NOTE - for NDO version > 4.2, this parameter is available only when external EPG is 
```

Comment:
Can you please check if the below implies the same meaning?

- In NDO versions over 4.2, the parameter is accessible only with an external EPG linked to the current schema-template's VRF.

Reply:
yes and changed